src/rpc/mod.rs

```diff
@@ -19,4 +23,14 @@ pub trait MercuryRpc {
 
     #[rpc(name = "is_in_rce_list")]
     fn is_in_rce_list(&self, rce_hash: H256, addr: H256) -> RpcResult<bool>;
+
+    #[rpc(name = "transfer_with_rce_completion")]
+    fn transfer_with_rce_completion(&self, transaction: Transaction) -> RpcResult<TransactionView>;
```

Comment:
Can we add some documentations showcasing how to use the added RPCs? Possibly in a doc file somewhere in the repo.